### PR TITLE
Controller prediction fix

### PR DESCRIPTION
**Pull request by zarik5**
Removed current prediction settings, added "Pose time offset" slider (basic setting). The new algorithm tries to make tracking consistent across different total latencies and framerates. Tracking starts to lag behind above 80ms of total latency and needs correction with Pose time offset.

### CODE REVIEW

File: alvr/server/cpp/alvr_server/alvr_server.cpp [hunk 1]
```diff
@@ -215,31 +214,37 @@ void DeinitializeStreaming() {
     // nothing to do
 }
 
+void SendVSync(float frameIntervalS) {
+    vr::Compositor_FrameTiming timings = {sizeof(vr::Compositor_FrameTiming)};
+    vr::VRServerDriverHost()->GetFrameTimings(&timings, 1);
+
+    // Warning: if the vsync offset deviates too much from 0, the latency starts to increase.
+    vr::VRServerDriverHost()->VsyncEvent(-frameIntervalS * timings.m_nNumVSyncsReadyForUse);
```
Comment (Vixea) [inline alvr/server/cpp/alvr_server/alvr_server.cpp:222]: Is this the call you were talking about that doesn't exist on Linux / or work?

Comment (zarik5) [reply alvr/server/cpp/alvr_server/alvr_server.cpp:222]: Yes

Comment (Vixea) [reply alvr/server/cpp/alvr_server/alvr_server.cpp:222]: Should it be guarded then or is it even to keep it this way

Comment (zarik5) [reply alvr/server/cpp/alvr_server/alvr_server.cpp:222]: It's already guarded. https://github.com/alvr-org/ALVR/pull/1299/files/8b07089306929570a1294c6cec5b2bcd5c818b10#diff-d643c72c21f7572f3441d4b6559f400b415a1956fe072a19eb832ae17b8bace5R769

**Comment by Vixea** [comment]
merging :D

**Pull request merged**